spec/models/survey_spec.rb

```diff
@@ -1,36 +1,95 @@
 require 'rails_helper'
 
-RSpec.describe Survey, type: :model do
-  it 'is valid with valid attributes' do
-    survey = Survey.new(question: 'Is this thing works?', start_date: Date.today, end_date: Date.tomorrow)
-    survey.survey_answers.new(answer: 'yes')
-    survey.survey_answers.new(answer: 'no')
-    expect(survey).to be_valid
-  end
+RSpec.describe Survey do
+  describe 'survey validation test' do
+    let(:user) { FactoryBot.create(:user, :admin) }
 
-  it 'is not valid with less than two answers' do
-    survey = Survey.new(question: 'Is this thing works?', start_date: Date.today, end_date: Date.tomorrow)
-    survey.survey_answers.new(answer: 'single answer')
-    expect(survey).to_not be_valid
-  end
+    it 'is valid with valid attributes' do
+      expect(user.surveys.new(
```

Comment:
it could be `let(:survey) { user.surveys.new(question: question....` and `let(:question) { 'Question?' }` so when you need test case with blank question you will just define `let(:question) { '' }` and will not touch or duplicate `survay`